storage/backup-restore/run-test.sh

```diff
@@ -56,7 +41,6 @@ mkdir ./node-data
 echo "Asserting node is empty"
 BACKUP_INTERVAL=0 docker compose up -d
 waitForDCService nodeA
-assertDiagnostic "http://localhost:11323" "node_did: \"\""
```

Comment:
no longer checks that node_did is empty because that would require a config, or removing/adding it to the config again. Can be deleted imo opinion since it was just an extra check to confirm that auto node did resolver could not find a DID.